protocol/streaming/full_node_streaming_manager.go

```diff
@@ -544,6 +718,14 @@ func (sm *FullNodeStreamingManagerImpl) InitializeNewStreams(
 				}
 
 				sm.SendSnapshot(allUpdates, subscriptionId, blockHeight, execMode)
```

Comment:
Since we have one stream, can we combine everything in just one snapshot function? Users might be confused they get snapshot messages in two chunks, even though they are for subaccounts and orderbooks.

Reply:
done